packages/identity/src/errors/TankerError.ts

```diff
@@ -1,8 +1,9 @@
-import { BaseError } from 'make-error';
-
-export class TankerError extends BaseError {
+export class TankerError extends Error {
   constructor(name: string = 'TankerError', message?: string) {
     super();
+
+    // Set the prototype explicitly.
```

Comment:
Maybe add a reference to a blog post/issue explaining why it is needed? (at least reference it in the base class, not necessarily everywhere).